162 2118749 proxy disconnected (#391)
infrawatch/documentation: doc-Service-Telemetry-Framework/assemblies/assembly_preparing-your-ocp-environment-for-stf.adoc

:context: assembly-preparing-your-ocp-environment-for-stf

[role="_abstract"]
To prepare your {OpenShift} environment for {Project} ({ProjectShort}), you must plan for persistent storage, adequate resources, event storage, and network considerations:

> * Ensure that persistent storage is available in your {OpenShift} cluster for a production grade deployment. For more information, see <<persistent-volumes_assembly-preparing-your-ocp-environment-for-stf>>.

Reviewer: ```suggestion
* Ensure that you have persistent storage available in your {OpenShift} cluster for a production grade deployment. For more information, see <<persistent-volumes_assembly-preparing-your-ocp-environment-for-stf>>.
```

> * Ensure that you have persistent storage available in your {OpenShift} cluster for a production grade deployment. For more information, see <<persistent-volumes_assembly-preparing-your-ocp-environment-for-stf>>.

Author: ```suggestion
* Ensure that you have persistent storage available in your {OpenShift} cluster for a production-grade deployment. For more information, see <<persistent-volumes_assembly-preparing-your-ocp-environment-for-stf>>.
```

* Ensure that enough resources are available to run the Operators and the application containers. For more information, see <<resource-allocation_assembly-preparing-your-ocp-environment-for-stf>>.
* Ensure that you have a fully connected network environment. For more information, see xref:con-network-considerations-for-service-telemetry-framework_assembly-preparing-your-ocp-environment-for-stf[].
ifeval::["{build}" == "upstream"]

> * {ProjectShort} uses ElasticSearch to store events, which requires a larger than normal `vm.max_map_count` value. The `vm.max_map_count` value is set by default in {OpenShift}. For more information about how to edit the value of `vm.max_map_count`, see <<node-tuning-operator_assembly-preparing-your-ocp-environment-for-stf>>.

Reviewer 2: ```suggestion
* {ProjectShort} uses Elasticsearch to store events, which requires a larger than normal `vm.max_map_count` value. The `vm.max_map_count` value is set by default in {OpenShift}. For more information about how to edit the value of `vm.max_map_count`, see <<node-tuning-operator_assembly-preparing-your-ocp-environment-for-stf>>.
```
